engine/src/flutter/impeller/renderer/backend/gles/render_pass_gles.cc

```diff
@@ -578,19 +578,23 @@ void RenderPassGLES::ResetGLState(const ProcTableGLES& gl) {
     // to discard the entire render target. Until we know the reason, default to
     // storing.
     bool angle_safe = gl.GetCapabilities()->IsANGLE() ? !is_default_fbo : true;
```

Comment:
`is_default_fbo` is just poorly named. Instead, I suggest renaming it to `is_wrapped_fbo` or something similar.

This rest of the change is accurate. Whether the FBO is wrapped of not, the type of enum to used to discard will depend on whether it is the default framebuffer object (and not if its wrapped).